cli/cmd/policy.go

```diff
@@ -442,9 +489,99 @@ func listPolicyTags(_ *cobra.Command, args []string) error {
 		return cli.OutputJSON(policyTagsResponse.Data)
 	}
 	if len(policyTagsResponse.Data) == 0 {
-		cli.OutputHuman("There were no policy tags found.")
+		cli.OutputHuman("There were no policy tags found.\n")
 		return nil
 	}
 	cli.OutputHuman(renderSimpleTable([]string{"Tag"}, policyTagsTable(policyTagsResponse.Data)))
 	return nil
 }
+
+func disablePolicy(_ *cobra.Command, args []string) error {
+	state := false
+	if len(args) > 0 {
+		policy := api.UpdatePolicy{PolicyID: args[0], Enabled: &state}
+		// todo: should alerts be disabled/enabled?
+		//policy.AlertEnabled = &policyState
+		_, err := cli.LwApi.V2.Policy.Update(policy)
+		if err != nil {
+			return err
+		}
+	}
+
+	// if tag is provided disable all policies matching
+	if policyCmdState.Tag != "" {
+		return setPolicyStateByTag(policyCmdState.Tag, state)
+	}
+	return nil
+}
+
+func enablePolicy(_ *cobra.Command, args []string) error {
+	state := true
+	if len(args) > 0 {
+		policy := api.UpdatePolicy{PolicyID: args[0], Enabled: &state}
+		// todo: should alerts be disabled/enabled?
```

Comment:
same, do we need a ticket?

Reply:
This was a question that was not certain when the PR was put up. It was left in intentionally until I was sure that we did not need alertEnabled set.

That question has been answered we can now remove this.